packages/components/dialog/src/token.ts

```diff
@@ -0,0 +1,14 @@
+import type { ComputedRef, CSSProperties, InjectionKey, Ref } from 'vue'
+import type { useNamespace } from '@element-plus/hooks'
+
+export type DialogContext = {
+  dialogRef: Ref<HTMLElement | null>
+  headerRef: Ref<HTMLElement | null>
+  ns: ReturnType<typeof useNamespace>
+  rendered: Ref<boolean>
+  style: ComputedRef<CSSProperties>
+}
+
+export const elDialogInjectionKey: InjectionKey<DialogContext> = Symbol(
+  'elDialogInjectionKey'
+)
```

Comment:
```suggestion
export const dialogInjectionKey: InjectionKey<DialogContext> = Symbol(
  'dialogInjectionKey'
)
```

Reply:
<img width="769" alt="image" src="https://user-images.githubusercontent.com/15975785/155076450-edbf9670-34d5-47fb-8f74-f059eee93441.png">

Reply:
<img width="672" alt="image" src="https://user-images.githubusercontent.com/15975785/155076551-ca5bbb94-ef76-4cb9-8a8a-53b1847deffd.png">

Reply:
I think in this case we should put the namespace to the symbol

Reply:
IMO, we should remove `el` prefix, like `scrollbar.ts`, `tabs.ts`, `config-provider.ts`